logo.js

```diff
@@ -76,6 +76,10 @@ function LogoInterpreter(turtle, stream, savehook)
     return atom === match;
   }
 
+  function isPromise(value) {
```

Comment:
Yeah. But I think we shouldn't need this function anyway, be async everywhere, and Promise.resolve() to cast anything that might not already be a promise. 

> On Jan 31, 2016, at 8:25 PM, Ian Bicking <notifications@github.com> wrote:
> 
> In logo.js:
> 
> > @@ -76,6 +76,10 @@ function LogoInterpreter(turtle, stream, savehook)
> >      return atom === match;
> >    }
> >  
> > +  function isPromise(value) {
> Yeah, I've seen the term, and yet it seems terribly obtuse doesn't it?
> 
> —
> Reply to this email directly or view it on GitHub.
>